pvr.iptvsimple/addon.xml.in

```diff
@@ -158,5 +158,55 @@
     <disclaimer lang="zh_CN">这是不稳定版的软件！作者不对录像失败、错误定时造成时间浪费或其它不良影响负责。</disclaimer>
     <disclaimer lang="zh_TW">這是測試中的軟體！原創作者無法針對以下情況負責：包括播放失敗，不正確的電子節目表，多餘的時數，或任何不可預期的不良影響。</disclaimer>
     <platform>@PLATFORM@</platform>
+    <news>
+v4.4.0
```

Comment:
Shouldn't this read v4.5.0? And where is the changelog for v4.4.0?

Reply:
:( 

I bet you this is in vuplus also.